app/views/katello/layouts/foreman_with_bastion.html.erb

```diff
@@ -2,4 +2,12 @@
   <%= render :partial => 'layouts/application_content' %>
 <% end %>
 
+<% content_for(:stylesheets) do %>
+  <%= stylesheet_link_tag *webpack_asset_paths('katello', :extension => 'css'), "data-turbolinks-track" => true %>
+<% end %>
+
+<% content_for(:javascripts) do %>
+  <%= javascript_include_tag *webpack_asset_paths('katello', :extension => 'js'), "data-turbolinks-track" => true, 'defer' => 'defer' %>
+<% end %>
+
```

Comment:
These changes are no longer required with the new approach in https://github.com/Katello/katello/pull/7032